Jetsnack/app/src/main/java/com/example/jetsnack/ui/JetsnackApp.kt

```diff
@@ -34,43 +35,39 @@ import com.example.jetsnack.ui.home.JetsnackBottomBar
 import com.example.jetsnack.ui.home.addHomeGraph
 import com.example.jetsnack.ui.snackdetail.SnackDetail
 import com.example.jetsnack.ui.theme.JetsnackTheme
-import com.google.accompanist.insets.ProvideWindowInsets
-import com.google.accompanist.insets.systemBarsPadding
 
 @Composable
 fun JetsnackApp() {
-    ProvideWindowInsets {
-        JetsnackTheme {
-            val appState = rememberJetsnackAppState()
-            JetsnackScaffold(
-                bottomBar = {
-                    if (appState.shouldShowBottomBar) {
-                        JetsnackBottomBar(
-                            tabs = appState.bottomBarTabs,
-                            currentRoute = appState.currentRoute!!,
-                            navigateToRoute = appState::navigateToBottomBarRoute
-                        )
-                    }
-                },
-                snackbarHost = {
-                    SnackbarHost(
-                        hostState = it,
-                        modifier = Modifier.systemBarsPadding(),
-                        snackbar = { snackbarData -> JetsnackSnackbar(snackbarData) }
-                    )
-                },
-                scaffoldState = appState.scaffoldState
-            ) { innerPaddingModifier ->
-                NavHost(
-                    navController = appState.navController,
-                    startDestination = MainDestinations.HOME_ROUTE,
-                    modifier = Modifier.padding(innerPaddingModifier)
-                ) {
-                    jetsnackNavGraph(
-                        onSnackSelected = appState::navigateToSnackDetail,
-                        upPress = appState::upPress
+    JetsnackTheme {
+        val appState = rememberJetsnackAppState()
+        JetsnackScaffold(
+            bottomBar = {
+                if (appState.shouldShowBottomBar) {
+                    JetsnackBottomBar(
+                        tabs = appState.bottomBarTabs,
+                        currentRoute = appState.currentRoute!!,
+                        navigateToRoute = appState::navigateToBottomBarRoute
                     )
                 }
+            },
+            snackbarHost = {
+                SnackbarHost(
+                    hostState = it,
+                    modifier = Modifier.systemBarsPadding(),
+                    snackbar = { snackbarData -> JetsnackSnackbar(snackbarData) }
+                )
+            },
+            scaffoldState = appState.scaffoldState
+        ) { innerPaddingModifier ->
+            NavHost(
+                navController = appState.navController,
+                startDestination = MainDestinations.HOME_ROUTE,
+                modifier = Modifier.padding(innerPaddingModifier)
```

Comment:
Just noticed while testing:

Jetsnack doesn't handle start and end navigation bar insets correctly, so content tends to be cut off. Don't want to necessarily hold up this conversion PR, but it'd be good to perhaps go through all insets usage across these samples to ensure there aren't other lingering issues.

Reply:
Yeah, that's a good idea, we can do a sample wide audit, thanks!